src/web/command_control/client/components/CommandControl.tsx

```diff
@@ -2823,6 +2862,12 @@ export default class CommandControl extends React.Component {
 				botInfo.botIdsDisconnected.push(bot?.bot_id)
 		    } else if (sendingDownload && !bot?.wifi_link_quality_percentage) {
 				botInfo.botIdsWifiDisconnected.push(bot?.bot_id)
+			} else if (sendingMission && pauseState.test(bot?.mission_state)) {
+				botInfo.botIdsInPauseState.push(bot?.bot_id)
+			} else if (sendingPause && (!stopAvailable.test(bot?.mission_state) || 
```

Comment:
I am pretty confused by this else if block. Maybe it needs a comment or just a quick explanation in reply to this msg